SequenceAnalysis/src/org/labkey/sequenceanalysis/query/SequenceAnalysisUserSchema.java

```diff
@@ -449,19 +446,20 @@ public DisplayColumn createRenderer(ColumnInfo colInfo)
             return new DataColumn(colInfo)
             {
                 @Override
-                public void renderGridCellContents(RenderContext ctx, Writer oldWriter, HtmlWriter out) throws IOException
+                public void renderGridCellContents(RenderContext ctx, HtmlWriter out)
                 {
                     String result = StringUtils.trimToNull(super.getFormattedHtml(ctx).toString());
-                    String delim = "";
+                    HtmlString delim = HtmlString.EMPTY_STRING;
                     if (result != null)
                     {
                         String[] tokens = result.split(",");
                         for (String token : tokens)
                         {
-                            String url = DetailsURL.fromString(_baseUrl + PageFlowUtil.encode(token), ctx.getContainer()).getActionURL().toString();
+                            ActionURL url = DetailsURL.fromString(_baseUrl + PageFlowUtil.encode(token), ctx.getContainer()).getActionURL();
 
-                            oldWriter.write(delim + "<a href=\"" + url + "\">" + token + "</a>");
-                            delim = "<br>";
+                            out.write(delim);
+                            out.write(PageFlowUtil.link(token).href(url));
```

Comment:
Another style change here. Not a big deal, but note that using `link(String, URLHelper)` is an even shorter option.